ansible/files/home/deployer/webhook/lib/deployer/app.rb

```diff
@@ -13,10 +14,38 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+require "openssl"
+require_relative "response"
+
 module Deployer
   class App
     def call(env)
-      [200, {}, ["Hello deployer"]]
+      request = Rack::Request.new(env)
+      response = Response.new
+      process(request, response) or response.finish
```

Comment:
必ず`finish`するならこれでよくない？

```suggestion
      process(request, response)
      response.finish
```

Reply:
fix: [0e38ab5](https://github.com/groonga/packages.groonga.org/pull/46/commits/0e38ab5c914e993ae9d12b1ad879500ac172a17b) 現状は必要ないのでシンプルな方に寄せておこうと思います。